### PR TITLE
Change any absolute imports to relative imports for consistency

### PR DESCRIPTION
**Pull request by astrofrog**
Closes https://github.com/vispy/vispy/issues/1218

### CODE REVIEW

File: vispy/app/backends/tests/test_ipynb_util.py [hunk 1]
```diff
@@ -3,10 +3,10 @@
 # Distributed under the (new) BSD License. See LICENSE.txt for more info.
 import numpy as np
 
-from vispy.app.backends._ipynb_util import (_extract_buffers,
+from ....app.backends._ipynb_util import (_extract_buffers,
```
Comment (Eric89GXL) [inline vispy/app/backends/tests/test_ipynb_util.py]: the test files are actually thought of more as scripts (note especially the `run_tests_if_main()` at the bottom that actually allows them to run if you call them) so all test script imports should be absolute

File: vispy/app/tests/test_backends.py [hunk 2]
```diff
@@ -10,12 +10,11 @@
...
-import vispy
-from vispy import keys
-from vispy.testing import (requires_application, assert_in, run_tests_if_main,
+from ... import app, keys
+from ...testing import (requires_application, assert_in, run_tests_if_main,
                            assert_raises)
-from vispy.app import use_app, Application
-from vispy.app.backends import _template
+from ...app import use_app, Application
+from ...app.backends import _template
```
Comment (Eric89GXL) [inline vispy/app/tests/test_backends.py]: These on the other hand *should* be relative if possible, so these changes are worthwhile

Comment (Eric89GXL) [inline vispy/app/tests/test_backends.py]: whoops, wrong line -- these should be absolute

File: vispy/app/backends/ipython/_widget.py [hunk 3]
```diff
@@ -13,8 +13,8 @@
     available, testable, why_not, which = False, False, str(exp), None
 else:
     available, testable, why_not, which = True, False, None, None
-from vispy.app.backends._ipynb_util import create_glir_message
-from vispy.app import Timer
+from ....app.backends._ipynb_util import create_glir_message
+from ....app import Timer
```
Comment (Eric89GXL) [inline vispy/app/backends/ipython/_widget.py]: These should be relative, as you have made them, if possible

File: vispy/app/backends/tests/test_ipynb_util.py (hunk 1 above)
Comment (astrofrog) [inline vispy/app/backends/tests/test_ipynb_util.py]: No problem - I'll undo the changes for all the test files

**Comment by Eric89GXL** [comment]
I am confused about the Travis error, @astrofrog can you reproduce locally?

**Comment by astrofrog** [comment]
@Eric89GXL - no, I've restarted the job though, so let's see if it was temporary.

**Comment by astrofrog** [comment]
@Eric89GXL - the failure here is just temporary (a download issue). Could you give me privileges on AppVeyor so I can restart issues like this?

**Comment by Eric89GXL** [comment]
Hopefully you have permission now, and I restarted the build already

File: vispy/util/wrappers.py [hunk 4]
```diff
@@ -79,20 +79,19 @@ def use(app=None, gl=None):
...
-        from vispy import config
+        from .. import gloo, config
         config['gl_backend'] = gl
-        vispy.gloo.gl.use_gl(gl)
+        gloo.gl.use_gl(gl)
     if app:
-        import vispy.app
-        vispy.app.use_app(app)
+        from .. import app
+        app.use_app(app)
```
Comment (Eric89GXL) [inline vispy/util/wrappers.py]: this is the problem -- this needs to be `from .. import app as vispy_app; vispy_app.use_app(app)`

Comment (Eric89GXL) [inline vispy/util/wrappers.py]: or better:
```
from ..app import use_app
use_app(app)
```

**Comment by astrofrog** [comment]
The continuum links seem breakable today...

**Comment by Eric89GXL** [comment]
Sometimes with AppVeyor I need to log out and back in, refresh pages, go to the project page, etc. to get the "restart" button

**Comment by Eric89GXL** [comment]
AppVeyor is being stupid in unrelated ways, merging

**Pull request merged**